enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/converter/MobileTokenConverter.java

```diff
@@ -39,6 +39,7 @@
 import java.math.BigDecimal;
 import java.util.*;
 
+import static com.wultra.security.powerauth.lib.mtoken.model.entity.PreApprovalScreen.ScreenType.QR_SCAN;
```

Comment:
We do not use static imports for other preapproval screen types, can we unify this?